README.md

```diff
@@ -55,33 +48,42 @@ If starting from a barcode and read file, you can first use the qiime [split_lib
 split_libraries_fastq.py -i XXX_R1_001.fastq -m map.txt -o split -b XXXX_I1_001.fastq -q 19
 ```
 
-and use the split/seqs.fna as the input to the deblur workflow.
+The resulting `split/seqs.fna` file can be used as the input to the deblur workflow.
+
+Input and Output Files
+======================
+
+The input to Deblur workflow is a directory of FASTA or FASTQ files (1 per sample) or a single demultiplexed FASTA or FASTQ file. These files can be gzip'd. The output directory will contain three BIOM tables in which the observation IDs are the Deblurred sequences. The outputs are contingent on the reference databases used and a more focused discussion on them is in the subsequent README section titled "Positive and Negative Filtering." The output files are as follows:
+
+- reference-hit.biom : contains only Deblurred reads matching the positive filtering database. By default, a reference composed of 16S sequences is used, and this resulting table will contain only those reads which recruit at a coarse level to it will be retained. Reads are also filtered against the negative reference, which by default will remove any read which appears to be PhiX or adapter.
+
+- reference-non-hit.biom : contains only Deblurred reads that did not align to the positive filtering database. Negative filtering is also appied to this table, so by default, PhiX and adapter are removed.
+
+- all.biom : contains all Deblurred reads. No filtering against the negative database is performed, meaning that by default, PhiX and adapter sequence will be present in this file. This file is a superset of the union of "reference-hit.biom", "reference-non-hit.biom" and all of the sequences which recruit to the negative reference database which is by default composed of PhiX and adapter.
```

Comment:
The all.biom does not contain the sequences aligning to the negative database (so does not contain phix reads). However, no positive filtering is performed (so contains 16s and non-16s sequences)